### ISSUE TITLE
Bindman-DNS support

### ISSUE THREAD

**Issue by tiagostutz** [go-acme/lego#729]
_[Note: this issue was edited because the product has changed the name]_

How about adding support to **Bindman-DNS**?
- https://github.com/labbsr0x/bindman-dns-webhook

**Bindman-DNS** is somewhat a REST-to-DNS solution. It has a REST spec of DNS operations and would bring a REST server exposing those operations. Each DNS provider can have its own implementation of the spec, so it receives the operations requests on a the same format but each one handles his own constraints and DNS update flow. On the other hand, the services that needs to make DNS changes and has Bindman-DNS on its stack doesn't need to know each provider spec because it will request this operation to Bindman-DNS and it will have the used DNS provider configured.

The scenario we are using **Bindman-DNS** is to listen for docker swarm services events that needs to trigger DNS updates but can be deployed in infrastructure with different DNS providers. (https://github.com/labbsr0x/bindman-dns-swarm-listener)

The first DNS provider available is *Bind9 DNS server* (https://github.com/labbsr0x/bindman-dns-bind9).

So, basically, the **Bindman-DNS** has two parts:
- *Manager*: the piece of software that actually interacts with the DNS provider server and knows its specifications. It implements a Hook (https://github.com/labbsr0x/bindman-dns-webhook) Spec that exposes REST operations of the Bindman-DNS spec. 
- *Client*: is the component that will invoke the Manager using the hook spec to update the DNS. Currently we have the *[docker swarm listener](https://github.com/labbsr0x/bindman-dns-swarm-listener)* implemented, that checks for *[traefik's](https://github.com/containous/traefik)* labels to grab the domains and invoke the DNS updates based on them.

**Comment by tiagostutz** [comment]
Ah, if you agree that this is a good feature we can code it and make a PR...
